README.md

```diff
@@ -47,7 +47,7 @@ so end-users have full guidance in using this tool-set and do not have to search
 
 First, start a Jenkins instance with JCasC [plugin](https://plugins.jenkins.io/configuration-as-code) installed.
 
-- Those running Jenkins as a [Docker container](https://github.com/jenkinsci/docker) (and maybe also [pre-installing plugins](https://github.com/jenkinsci/docker#preinstalling-plugins)), do include [configuration-as-code](https://plugins.jenkins.io/configuration-as-code) plugin.
+- Those running Jenkins as a [Docker container](https://github.com/jenkinsci/docker) (and maybe also [pre-installing plugins](https://github.com/jenkinsci/docker#preinstalling-plugins)), do include [configuration-as-code](https://plugins.jenkins.io/configuration-as-code) plugin and optional the [configuration-as-code-support](https://plugins.jenkins.io/configuration-as-code-support) if you use one of the credentials plugins or the job-dsl (see link).
```

Comment:
Missing the link
I think `optionally` should be used instead of optional.
Potentially a README should be added to the support plugin section here: https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/support

That should detail all the plugins that the support plugin gives support for

This should definitely be added to the README as its currently missing